vms/platformvm/vm_test.go

```diff
@@ -277,13 +277,14 @@ func defaultVM(t *testing.T, fork activeFork) (*VM, database.Database, *mutableS
 		return nil
 	}
 
+	dynamicConfigBytes := []byte(`{"network":{"max-validator-set-staleness":0}}`)
```

Comment:
There might be other configs that would be good to default to. But this disables the optimization so that we re-calculate the validator set aggressively.